fs2-aws-kinesis/src/main/scala/fs2/aws/kinesis/Kinesis.scala

```diff
@@ -155,14 +155,17 @@ object Kinesis {
         case _ => ()
       }
 
-      retrievalConfig.initialPositionInStreamExtended(
-        settings.initialPositionInStream match {
-          case Left(position) =>
-            InitialPositionInStreamExtended.newInitialPosition(position)
-
-          case Right(date) =>
-            InitialPositionInStreamExtended.newInitialPositionAtTimestamp(date)
-        }
+      retrievalConfig.streamTracker(
+        new SingleStreamTracker(
+          StreamIdentifier.singleStreamInstance(settings.streamName),
+          settings.initialPositionInStream match {
+            case Left(position) =>
+              InitialPositionInStreamExtended.newInitialPosition(position)
+
+            case Right(date) =>
+              InitialPositionInStreamExtended.newInitialPositionAtTimestamp(date)
+          }
+        )
```

Comment:
unsure if this is correct, but `initialPositionInStreamExtended` is deprecated :/